control/velocity_controller/include/velocity_controller/velocity_controller.hpp

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+#include <geometry_msgs/msg/pose_with_covariance_stamped.hpp>
+#include <geometry_msgs/msg/twist_with_covariance_stamped.hpp>
+#include <geometry_msgs/msg/wrench_stamped.hpp>
+#include <rclcpp/rclcpp.hpp>
+#include <std_msgs/msg/bool.hpp>
+#include <std_msgs/msg/float64_multi_array.hpp>
+#include <std_msgs/msg/string.hpp>
+#include "LQR_setup.hpp"
+#include "nav_msgs/msg/odometry.hpp"
+#include "velocity_controller/PID_setup.hpp"
+#include "vortex_msgs/msg/los_guidance.hpp"
+
+class Velocity_node : public rclcpp::Node {
+   public:
```

Comment:
Goes for all the classes here:

If you use the `class` keyword, you are signalling to the reader / user / yourself that you have a clear distinction between publicly exposed functionality and everything else (private). 

Good litmus test to ask yourself "do I need to expose X to the user of this class". If the answer is no, make it private. (I think you will find a few cases like that here).

Refer back to the C++ slides regarding encapsulation for the reasoning why you'd want to do that.